autoload/slime/targets/tmux.vim

```diff
@@ -24,8 +24,8 @@ function! slime#targets#tmux#send(config, text)
   " reasonable hardcode, will become config if needed
   let chunk_size = 1000
 
-  for i in range(0, len(text_to_paste) / chunk_size)
-    let chunk = text_to_paste[i * chunk_size : (i + 1) * chunk_size - 1]
+  for i in range(0, strchars(text_to_paste) / chunk_size)
+    let chunk = strcharpart(text_to_paste, i * chunk_size, chunk_size)
```

Comment:
oooh, I like it ^

is there a chance this `strcharpart` could be over the byte limit and end up truncated? 🤔

Reply:
I've only tested it up to ~O(1M) characters for very large prompts, haven't seen any evidence of truncation.

Reply:
I can _imagine_ edge cases involving heavy multibytes text (e.g. Japanese), but we can tackle that if/when it happens

depending how what we discover, I might cc you in a ticket (if that's OK)